patsicko/my-brand-bakend
# Ft upload image

## What this PR do
Validate blog posts and user creation,
add image upload feature
## Description of the tasks to be completed
-  Validation of blog posts and user using joi
-  upload image using cloudinary and multer
[Finishes#]

## How should this be manually tested
- open postman
- set your HTTP request to POST
- input API endpoint
- switch to the Body tab
- Click raw
- select JSON
-  create a json object with blogTitle, blogAutor,blogImage and blogText to post and skip password
- Click Send.
- Status: 400Bad Request should be displayed
- {
    "status": "error",
    "message": "\"password\" is required"
}
